src/admin/python/lsst/qserv/admin/itest_load.py

```diff
@@ -73,6 +73,11 @@ def execute(cursor: MySQLCursor, stmt: str, multi: bool = False) -> None:
         cursors = [cursor]
 
     for cursor in cursors:
+        # Cast here because MySQLCursorAbtract does not have with_rows for some reason, even though both of
+        # its subclasses do...
```

Comment:
Could be an oversight, you can probably complain to mysql folks about it.

Reply:
I'll file an issue.